index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="top-head">
+        <ul class="top-desk">
+        <li><img class="fb" src="asset/facebook.png" alt="facebook"></li>
+        <li><img class="twit" src="asset/twitter.png" alt="twitter"></li>
+        <li>English</li>
+        <li>My page</li>
+        <li>Log out</li>
+        </ul>
+    </div>
+    <div class="desk-menu">
+        <img class="mic-logo" src="asset/mic logo.png">
+        <ul class="desk-list">
+            <li onclick="openmenu()" class="ab-desk"><a href="./about.html">About</a></li>
+            <li onclick="openmenu()" class=""><a href="#pro">Program</a></li>
+            <li onclick="openmenu()"><a href="#"></a>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+            <li class="desk-cc">CC Campaign</li>
+        </ul>
+    </div>
+    <div class="toggle">
+        <img onclick="openMenu()" id="hamburger"  src="asset/mobile_menu 1.png">
+        <img class="mic-log" src="asset/mic logo.png">
+    </div>
+    <div class="menu-mob hidden" id="mobile-menu">
+        <img  onclick="closeMenu()" class="closing" src="asset/Icon - Cancel.png" id="closing-icon">
+
+        <ul class="menu-list" id="hum-list">
+            <li onclick="openMenu()" class=""><a class="" href="./about.html">About</a></li>
+            <li onclick="openmenu()" class=""><a class="" href="#program"></a>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+            <li>CC Campaign</li>
+        </ul>
+    </div>
+    </header>
+    <section class="main-section">
```

Comment:
- [ ] please kindly make your main title bigger like that of the original design. 
![Screenshot (1293)_LI](https://github.com/Surafels/Capstone_Project/assets/65298048/8087a2d2-3138-49fb-a5e3-389b1b846753)